Cargo.toml

```diff
@@ -11,16 +11,16 @@ repository = "https://github.com/rustls/rustls-native-certs"
 categories = ["network-programming", "cryptography"]
 
 [dependencies]
-rustls-pemfile = "=2.0.0-alpha.2"
-pki-types = { package = "rustls-pki-types", version = "0.2.2" }
+rustls-pemfile = "2.0.0"
```

Comment:
I think we usually prefer to admit trailing zero components.

Reply:
I removed the existing instances in an initial commit, and then added updates that adhere to this style afterwards.